Tests/LinuxMain.swift

```diff
@@ -5,22 +5,3 @@
 //  Created by Valerio Mazzeo on 27/11/2018.
 //  Copyright © 2018 Asensei Inc. All rights reserved.
 //
-
```

Comment:
I've been adding the line ```// The LinuxMain.swift file is required to run `swift build` without the `--enable-test-discovery``` flag.` so that it's clear why an empty is file is there.
```suggestion
// The LinuxMain.swift file is required to run `swift build` without the `--enable-test-discovery` flag.
```